package.json

```diff
@@ -79,6 +79,8 @@
     "fixyarn":"git checkout origin/master -- yarn.lock && yarn install && git add yarn.lock"
   },
   "dependencies": {
+    "@types/bootstrap": "^3.3.36",
```

Comment:
btw it should be --dev dependency.
(shameless plug) Use [tadd](https://github.com/krzkaczor/tadd) to install them quickly xD